### ISSUE TITLE
Additional payload options

### ISSUE THREAD

**Issue by farski** [clonn/slack-node-sdk#22]
There are some additional parameters that can be sent along with the payload (`link_names`, `mrkdwn`, etc), that currently get stripped out because the payload is being built explicitly. Would it be possible to make it so those options could get passed along?

**Comment by kevinburkeshyp** [comment]
hey! I believe you can do this... doesn't https://github.com/clonn/slack-node-sdk/blob/master/src/lib/slack.seed.coffee#L77 set any options you pass through on the request?

**Comment by farski** [comment]
@kevinburkeshyp I think maybe that works with the API but no webhooks, because only certain keys are being added to the payload https://github.com/clonn/slack-node-sdk/blob/master/src/lib/slack.seed.coffee#L36

**Comment by strix** [comment]
I agree with @farski.  In my case, I want to override the `unfurl_links` option.

**Comment by jasiedu** [comment]
My emojis are not getting sent. Is this a config thing?

**Comment by simonrand** [comment]
+1 from me also, for now I've forked and added these options (https://github.com/clonn/slack-node-sdk/compare/master...simonrand:allow_unfurl_options) but would be nice to have this upstream..
